misk-hibernate/src/main/kotlin/misk/hibernate/RealTransacter.kt

```diff
@@ -129,6 +143,12 @@ internal class RealTransacter private constructor(
     val retryJitterMillis: Long = 400
   )
 
+  companion object {
+    val APPLICATION_TRANSACTION_SPAN_NAME = "app-db-transaction"
+    val DB_TRANSACTION_SPAN_NAME = "db-session"
+    val TRANSACTER_SPAN_TAG = "hibernate-transacter"
```

Comment:
just for my knowledge, what does this do for you?

Reply:
just another tag, referenced as "component". A bit redundant, we can remove it down the line if we don't find it useful. From the docs:

"COMPONENT is a low-cardinality identifier of the module, library, or package that is instrumented."